src/rusoto/mod.rs

```diff
@@ -334,3 +334,16 @@ impl From<Option<SignedRequestPayload>> for RusotoBody {
         RusotoBody { inner }
     }
 }
+
+pub fn is_retriable_error<T>(error: &RusotoError<T>) -> bool {
```

Comment:
This seems like it would be better as:
```rust
pub impl RusotoError<T> {
    pub fn is_retriable_error(&self) -> bool;
}
```

Reply:
Is this okay to extend crate? (in case of `vector` can be crate itself)

Reply:
No, you're right, that's an external type so we can't impl for it. I misread this as for the `AwsRusotoError`.